src/test/java/reposense/util/TestUtil.java

```diff
@@ -133,6 +135,27 @@ public static void compareAuthor(Author expectedAuthor, Author actualAuthor) {
         Assert.assertEquals(expectedAuthor.getAuthorAliases(), actualAuthor.getAuthorAliases());
     }
 
+    /**
+     * Compares attributes of {@code expectedFormats} and the white-listed formats in {@code actualFileTypeManager}.
+     *
+     * @throws AssertionError if the formats in {@code actualFileTypeManager} is different from
+     * {@code expectedFormats}.
+     */
+    public static void compareWhitelistedFormats(
+            List<FileType> expectedFormats, FileTypeManager actualFileTypeManager) {
+        Assert.assertEquals(expectedFormats, actualFileTypeManager.getFormats());
```

Comment:
A bit unnecessary from my point for these two methods. 
Can directly use `Assert.assertEquals(TEST_REPO_BETA_CONFIG_FORMATS, config.getFileTypeManager().getFormats)` 

Can ask seniors opinion.

Reply:
yes, indeed quite redundant

Reply:
Have removed those